Parser/pegen.c

```diff
@@ -391,6 +391,21 @@ _PyPegen_raise_error_known_location(Parser *p, PyObject *errtype,
     PyObject *tmp = NULL;
     p->error_indicator = 1;
 
+    if (p->start_rule == Py_fstring_input) {
+        const char *fstring_msg = "f-string: ";
+        Py_ssize_t len = strlen(fstring_msg) + strlen(errmsg);
+
+        char *new_errmsg = PyMem_RawMalloc(len + 1); // Lengths of both strings plus NULL character
```

Comment:
This should be `PyMem_Malloc` no?

Reply:
I used `PyMem_RawMalloc`, because `fstring_compile_expr` uses it for `str` in `string_parser.c`. You know best what's more suitable here.

Reply:
I would like to move them to `PyMem_Malloc`, but let's do that in a different PR